### ISSUE TITLE
WaveNet as a Virtual instrument

### ISSUE THREAD

**Issue by henrycornish** [r9y9/wavenet_vocoder#141]
Hi Guys,

I'm currently working to adapt this project to work as a virtual instrument using
midi vectors as local conditioning as outlined here: 
https://arxiv.org/abs/1806.09905 
https://arxiv.org/abs/1810.12247

I'm using Solo Cello recordings from the musicNet dataset
and have written a pre-processing script to generate the conditioning vectors.

I have run into a couple issues getting the wavenet too train. I am at 20K steps
with an avg loss of 6 (tensorflow log attached) and am getting noise.

I'm worried that this is due to the fact the data is only split across 10 files of about 5 mins long each so it goes through epochs very quickly, this is certainly slowing down training considerably. 

the audio printed to checkpoints\audio\stepXXXX_predicted seems to sound good though. am I missing something here?

I'm doing this for a piece of university research, however, the deadline is in a few weeks so i would be very grateful if anyone has any comments on this.

![logs-solocello](https://user-images.githubusercontent.com/25873176/52244082-e6637900-28d3-11e9-8617-9e8e11656966.PNG)
![midi_vector](https://user-images.githubusercontent.com/25873176/52244083-e6637900-28d3-11e9-837c-ed7f04cce984.png)
![train_eval_step000020000_waveplots](https://user-images.githubusercontent.com/25873176/52244084-e6fc0f80-28d3-11e9-914d-3ef1527b7725.png)
